### PR TITLE
Added LR syntax error recovery paper

### PR DESCRIPTION
**Pull request by ptersilie**

### CODE REVIEW

**Comment by ptersilie** [comment]
Ready for re-review

**Comment by ptersilie** [comment]
Ready for re-review again

File: softdev.bib [hunk 1]
```diff
@@ -1652,3 +1652,13 @@ @manual{intel17pstate
...
 }
+
+@article{corchuelo02repairing,
+    author = {Corchuelo, Rafael and Antonio Pérez, José and Ruiz-Cortés, Antonio and Toro, Miguel},
+    year = {2002},
+    month = Nov,
+    pages = {698-710},
+    title = {Repairing syntax errors in LR parsers},
+    volume = {24},
+    booktitle = {ACM Trans. Program. Lang. Syst.}
```
Comment (ltratt) [inline softdev.bib]: Should probably be `TOPLAS`.

**Comment by ltratt** [comment]
OK, if you squash I can merge.

**Pull request merged**